### PR TITLE
www: update links to 2.0

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

File: cite.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h1>
 
             <p><b>1. Body text</b>
 	      <pre><code>
-Computations were done using the ASPECT code version 1.5.0, see \cite{heister_aspect_methods2,...}.</code></pre>
+Computations were done using the ASPECT code version 2.0.0, see \cite{heister_aspect_methods2,...}.</code></pre>
```
Comment (bangerth) [inline cite.html]: We should really remove the `,...` here. That's just lazy.

File: header.include [hunk 2]
```diff
@@ -40,8 +40,9 @@
               <a href="#" class="dropdown-toggle" data-toggle="dropdown">Development<b class="caret"></b></a>
               <ul class="dropdown-menu">
                 <li><a href="../doc/doxygen/index.html">Documentation</a></li>
-                <li><a href="../doc/doxygen/changes_current.html">Changes after v1.5.0</a></li>
+                <li><a href="../doc/doxygen/changes_current.html">Changes after v2.0.0</a></li>
 
+		<li><a href="../doc/doxygen/changes_between_1_85_80_and_2_80_80.html">Changes v1.5&#10142;2.0</a></li>
```
Comment (bangerth) [inline header.include]: missing space for alignment

Comment (tjhei) [reply header.include]: fixed

File: cite.html (hunk 1 above)
Comment (tjhei) [reply cite.html]: This is there on purpose. Can we discuss maybe during the hackathon?

Comment (bangerth) [reply cite.html]: Yes. I think I know what you mean, though if you hope that a user puts something specific into these dots, then you should say what exactly right here. (Unless that comes next -- github just shows me this one line right now.)

**Pull request merged**